PMMain/Midas/MidasGUI/RingData.h

```diff
@@ -0,0 +1,34 @@
+#include <vector>
+#pragma once
+class ringData
+{
+public:
+    ringData();
+    ~ringData();
+
+    void Dispose_object()
```

Comment:
Your naming convention is inconsistent. Some of your methods use camelcase (i.e. thisIsAFunction() ), and some use the underscores (this_is_a_function()). We've been using camelcase for everything else, so you should change your names to reflect that.